parts/linux/cloud-init/artifacts/update_certs_provisioning.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Adds certificates passed in managed cluster body during node provisioning in AKS
+
+[Service]
+Type=oneshot
+ExecStart=/opt/scripts/update_certs.sh
+ExecStart=/opt/scripts/restart_containerd.sh
```

Comment:
do you really want this? it will trigger the restart every time the certs on disk update, including from DS.

probably the right thing for users but a bit of a behavior change for people who didn't need this specific functionality